src/styles/index.scss

```diff
@@ -1,3 +1,18 @@
+$day-size: 100px;
+$days: ('mon' 0, 'tue' 1, 'wed' 2, 'thu' 3, 'fri' 4, 'sat' 5, 'sun' 6);
+$gap: 1px;
+$padding: 10px;
+
+@import './main';
```

Comment:
Consider adding a new line before the import statement for better readability and separation of concerns. [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)